crates/forge/tests/cli/script.rs

```diff
@@ -975,7 +961,10 @@ forgetest_async!(check_broadcast_log, |prj, cmd| {
 
     // Prepare CREATE2 Deployer
     let addr = address!("0x4e59b44847b379578588920ca78fbf26c0b4956c");
-    let code = hex::decode("7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf3").expect("Could not decode create2 deployer init_code").into();
+    let code =
+hex::decode("
+7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf3"
+).expect("Could not decode create2 deployer init_code").into();
```

Comment:
To do: avoid formatting changes introduced here due to commenting out tests and restoring